pyclam/manifold.py

```diff
@@ -1342,7 +1333,7 @@ def dump(self, fp: Union[BinaryIO, IO[bytes]]) -> None:
         pickle.dump({
             'metric': self.metric,
             'root': self.root.json(),
-            'graph': [cluster.name for cluster in self.graph.clusters]
+            'graphs': [[cluster.name for cluster in graph.clusters] for graph in self.graphs]
```

Comment:
Consider a set for the inner comprehension

Reply:
done